photos.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: main
+permalink: photos.html
+---
+
+# 2WebsitE - Photo Gallery
+
+## PHOTOS!!!
+
+{% for image in site.static_files %}
+
+{% if image.path contains 'img/gallery' %}
+
+![{{ image.basename }}]({{ site.url }}{{ image.path }}){:.img-thumbnail}
```

Comment:
`{{ site.url }}` can safely be removed, I believe. The parameter just equates to a blank string since it's not set in `_config.yml` anyways, and this site should be deployable regardless of what domain it's sitting on.

I looked into why the hover text was gone, and it turns out that the thing being set here, the `alt` text, is actually different from the `title` text, which is what we want. Title text syntax is `![alt text](url "title text"){:css identifiers}`.

Reply:
Updated!